read-the-docs/source/ServiceNow.rst

```diff
@@ -0,0 +1,294 @@
+Servicenow
+==========
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets is included.
+
+Note: Because each instance of ServiceNow can have custom fields and
+custom values, some of the tested fields may not be applicable to
+certain instances of ServiceNow. Additionally, your ServiceNow instance
+may contain ticket fields that we have not tested. Keep in mind
+different ServiceNow tables contain different fields with different
+properties. Mandatory fields in one table may be optional or
+non-existent in another table.
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST
+API right afterwards. You can also use
+`get\_ticket\_content <#content>`__ to get ticket content including
+preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki: -
+`REST API <http://wiki.servicenow.com/index.php?title=REST_API>`__ -
```

Comment:
These three links got jumbled together on readthedocs. They should be in a bulleted list as they are in: https://github.com/dmranck/ticketutil/blob/master/docs/servicenow.md